README.md

```diff
@@ -25,6 +25,7 @@ canonical way to set up a metalkube cluster.
 - ideally on a bare metal host
 - run as a user with passwordless sudo access
 - get a valid pull secret (json string) from https://cloud.openshift.com/clusters/install#pull-secret
+- hostnames for masters and workers must be in the format XX-master-# (e.g. openshift-master-0), or XX-worker-# (e.g. openshift-worker-0)
```

Comment:
Will it work with FQDN, e.g: openshift-master-0.qe1.kni.lab.eng.bos.redhat.com ?

Reply:
I believe as long as qe1.kni.lab.eng.bos.redhat.com is the $CLUSTER_DOMAIN, this is how the VM environment looks:

```console
$ hostname -f
master-0.ostest.test.metalkube.org
```

Reply:
it does work with XX-master-0 and so on